src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,27 +308,42 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		const editor = this._notebookService.findNotebookEditor(context);
 		// Set defaultParameters to the parameter values in parameter cell
 		let defaultParameters = new Map<string, string>();
+		let emptyParameterCell: boolean;
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
+				// Check if cell is empty or contains just whitespace
+				const cellSource = typeof cell.source === 'string' ? [cell.source] : cell.source;
+				for (let x of cellSource) {
+					emptyParameterCell = /^\s*$/.test(x);
+				}
+				if (emptyParameterCell) {
+					// If there is no parameters in the cell indicate to user to add them
+					this.notificationService.notify({
+						severity: Severity.Info,
+						message: localize('noParametersInCell', "This notebook cannot run with parameters until there are parameters added to the parameter cell. [Learn more](https://docs.microsoft.com/sql/azure-data-studio/notebooks/notebooks-parameterization)."),
+					});
+					return;
+				}
 				for (let parameter of cell.source) {
 					let param = parameter.split('=', 2);
```

Comment:
You're assuming that the line is going to have a = in it. If it doesn't though it'll throw an error (null ref since param[1] is undefined)

If you're going to be assuming this then you should likely want to test that as well. You could add it to the empty-line check above if you wanted to consolidate the checks.

Reply:
I think documentation will be clear to help users understand that '=' is needed for parameters, and only those values would be the ones able to be changed inside the parameter cell.